backtrace-library/src/main/java/backtraceio/library/breadcrumbs/BacktraceBreadcrumbsLogger.java

```diff
@@ -0,0 +1,86 @@
+package backtraceio.library.breadcrumbs;
+
+import com.squareup.tape.QueueFile;
```

Comment:
Maybe run optimize imports on whole project - you will save your time and fix in all places

Reply:
Will do, thank you!